authorization/invitation/service/invitation_service.go

```diff
@@ -162,3 +211,98 @@ func (s *invitationServiceImpl) Issue(ctx context.Context, issuingUserId uuid.UU
 
 	return nil
 }
+
+type invitationNotification struct {
+	invitation *invitationrepo.Invitation
+	roles      []string
+}
+
+// processTeamInviteNotifications sends an e-mail notification to a user.
+func (s *invitationServiceImpl) processTeamInviteNotifications(ctx context.Context, team *account.Identity, inviterName string,
+	notifications []invitationNotification, witURL string) error {
+	teamName := team.IdentityResource.Name
+
+	var spaceName string
+	var err error
+
+	if witURL != "" {
+		spaceName, err = lookupSpaceName(ctx, witURL, *team.IdentityResource.ParentResourceID)
+		if err != nil {
+			return err
+		}
+	}
+
+	var messages []notification.Message
+
+	for _, n := range notifications {
+		acceptURL := fmt.Sprintf("%s/api/invitations/accept?code=%s", s.config.GetAuthServiceURL(), n.invitation.AcceptCode.String())
+
+		messages = append(messages, notification.NewTeamInvitationEmail(n.invitation.Identity.UserID.UUID.String(),
+			teamName,
+			inviterName,
+			spaceName,
+			acceptURL))
+	}
+
+	s.Services().NotificationService().SendMessagesAsync(ctx, messages)
+
+	return nil
```

Comment:
return s.Services().NotificationService().SendMessagesAsync(ctx, messages)
instead of 
return nil
Otherwise you will miss an error returned by notification service.

Reply:
Fixed